[CCE] Node creation fails with panic

Found on the latest `devel` running `TestAccCCEClusterV3DataSource_basic`:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x290 pc=0xf09172]

goroutine 1803 [running]:
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/cce.resourceCCENodeV3Create({0x1707cb8, 0xc000f0ede0}, 0xc000c437b8, {0x1465e80, 0xc0006601c0})
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/cce/resource_opentelekomcloud_cce_node_v3.go:540 +0xff2
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000209b20, {0x1707c48, 0xc00069dc00}, 0x2, {0x1465e80, 0xc0006601c0})
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:330 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000209b20, {0x1707c48, 0xc00069dc00}, 0xc000218770, 0xc000e562a0, {0x1465e80, 0xc0006601c0})
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:456 +0x871
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc0001372c0, {0x1707c48, 0xc00069dc00}, 0xc0000b4f00)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:955 +0x9aa
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ApplyResourceChange(0xc0009111e0, {0x1707cf0, 0xc000e33650}, 0x4fb)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:332 +0x6c
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x140f7c0, 0xc0009111e0}, {0x1707cf0, 0xc000e33650}, 0xc000f0e360, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0004b7dc0, {0x1720fb0, 0xc001134180}, 0xc000e2ef00, 0xc00063c150, 0x2060160, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1194 +0xc8f
google.golang.org/grpc.(*Server).handleStream(0xc0004b7dc0, {0x1720fb0, 0xc001134180}, 0xc000e2ef00, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1517 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x294

Process finished with the exit code 1
```

Part of #1288

--- a/opentelekomcloud/services/cce/resource_opentelekomcloud_cce_node_v3.go
+++ b/opentelekomcloud/services/cce/resource_opentelekomcloud_cce_node_v3.go
@@ -522,11 +522,10 @@ func resourceCCENodeV3Create(ctx context.Context, d *schema.ResourceData, meta i
 	}
 
 	log.Printf("[DEBUG] Create Options: %#v", createOpts)
-	_, err = nodes.Create(client, clusterID, createOpts).Extract()
-	var node *nodes.Nodes
+	node, err := nodes.Create(client, clusterID, createOpts).Extract()
 	switch err.(type) {
 	case golangsdk.ErrDefault403:
-		retryNode, err := recursiveCreate(ctx, client, createOpts, clusterID, 403)
+		retryNode, err := recursiveCreate(ctx, client, createOpts, clusterID)
 		if err == "fail" {
 			return fmterr.Errorf("error creating OpenTelekomCloud Node")
 		}
@@ -999,29 +998,24 @@ func waitForClusterAvailable(cceClient *golangsdk.ServiceClient, clusterId strin
 	}
 }
 
-func recursiveCreate(ctx context.Context, client *golangsdk.ServiceClient, opts nodes.CreateOptsBuilder, clusterID string, errCode int) (*nodes.Nodes, string) {
-	if errCode == 403 {
-		stateCluster := &resource.StateChangeConf{
-			Target:     []string{"Available"},
-			Refresh:    waitForClusterAvailable(client, clusterID),
-			Timeout:    15 * time.Minute,
-			Delay:      15 * time.Second,
-			MinTimeout: 3 * time.Second,
-		}
-		_, stateErr := stateCluster.WaitForStateContext(ctx)
-		if stateErr != nil {
-			log.Printf("[INFO] Cluster Unavailable %s.\n", stateErr)
-		}
-		s, err := nodes.Create(client, clusterID, opts).Extract()
-		if err != nil {
-			if _, ok := err.(golangsdk.ErrDefault403); ok {
-				return recursiveCreate(ctx, client, opts, clusterID, 403)
-			} else {
-				return s, "fail"
-			}
-		} else {
-			return s, "success"
-		}
+func recursiveCreate(ctx context.Context, client *golangsdk.ServiceClient, opts nodes.CreateOptsBuilder, clusterID string) (*nodes.Nodes, string) {
+	stateCluster := &resource.StateChangeConf{
+		Target:     []string{"Available"},
+		Refresh:    waitForClusterAvailable(client, clusterID),
+		Timeout:    15 * time.Minute,
+		Delay:      15 * time.Second,
+		MinTimeout: 3 * time.Second,
 	}
-	return nil, "fail"
+	_, stateErr := stateCluster.WaitForStateContext(ctx)
+	if stateErr != nil {
+		log.Printf("[INFO] Cluster Unavailable %s.\n", stateErr)
+	}
+	node, err := nodes.Create(client, clusterID, opts).Extract()
+	if err != nil {
+		if _, ok := err.(golangsdk.ErrDefault403); ok {
+			return recursiveCreate(ctx, client, opts, clusterID)
+		}
+		return node, "fail"
+	}
+	return node, "success"
 }

--- a/releasenotes/notes/cce-node-736f0379edfbe7af.yaml
+++ b/releasenotes/notes/cce-node-736f0379edfbe7af.yaml
@@ -1,0 +1,4 @@
+---
+fixes:
+  - |
+    **[CCE]** Fix panic at ``resource/opentelekomcloud_cce_node_v3`` creation (`#1319 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1319>`_)

